e2e/cli/command/run.go

```diff
@@ -202,6 +201,11 @@ func (c *Run) runTest(opts *runOpts) (*TestReport, error) {
 		return nil, err
 	}
 
+	err = cmd.Wait()
+	if err != nil {
+		return nil, err
```

Comment:
Calling `Wait` is good here, though we probably want to suppress (and log?!) its errors.  When a test execution fails and get a non-zero exit code, the TestReport will contain more useful info about failed tests and the like.

I wonder if that's why Wait wasn't called before.

Reply:
I took a swing at this, but there are a lot of logging options used in nearby code. Does this count as suppressed and logged?

Reply:
I was thinking of something more like, where we don't return from function immediately and still return test report.

```
	err = cmd.Wait()
	if err != nil {
		// when command fail, log here then proceed to generate test report
		// to report more informative info about which tests fail
		c.logger.Error("test command failed", "error", err)
	}
```

Reply:
I made this change, nearly verbatim.